src/pkg/count/count.go

```diff
@@ -12,7 +12,7 @@ type Bus struct {
 
 func New() *Bus {
 	return &Bus{
-		stats: xsync.NewMapOf[*xsync.Counter](),
+		stats: xsync.NewMapOf[string, *xsync.Counter](),
```

Comment:
Why are we not using `syncd` here?

Reply:
primarily because I need to pull in xsync for the counter anyway.  Personally I think its an acceptable caveat to the centralized ownership.  Do you think that'll cause excess confusion?